perses/tests/test_samplers.py

```diff
@@ -154,10 +154,187 @@ def run_samplers(testsystem_names, niterations=5):
 #        f.description = "Testing expanded ensemble sampler with AlanineDipeptideTestSystem '%s'" % environment
 #        yield f
 
+def test_RESTCapableHybridTopologyFactory_repex_neutral_mutation():
+    """
+    Run ALA->THR and THR->ALA repex with the RESTCapableHybridTopologyFactory and make sure that the free energies are equal and opposite.
+
+    """
+
+    import tempfile
+    from pkg_resources import resource_filename
+    import numpy as np
+
+    from openmm import unit
+
+    from perses.app.relative_point_mutation_setup import PointMutationExecutor
+    from perses.dispersed.utils import configure_platform
+    from perses.samplers.multistate import HybridRepexSampler
+
+    from openmmtools.multistate import MultiStateReporter, MultiStateSamplerAnalyzer
+    from openmmtools import cache, utils, mcmc
+    platform = configure_platform(utils.get_fastest_platform().getName())
+    cache.global_context_cache.platform = platform
```

Comment:
We shouldn't try to change the `global_context_cache` platform this way, since already by default it should be using the fastest available platform. Also, as far as I see we are specifying local contexts in lines 221 and 222, so the `global_context_cache` is not used.  I think the tests are failing because the `global_context_cache` is imported and being used in other parts of the code already. Removing line 176 should fix this.
```suggestion
    platform = configure_platform(utils.get_fastest_platform().getName())
```